modules/player_pc_navigaton.py

```diff
@@ -0,0 +1,181 @@
+from modules.context import context
+from modules.game import decode_string
+from modules.memory import read_symbol, unpack_uint16
+from modules.menuing import BaseMenuNavigator, scroll_to_item_in_bag
+from modules.items import Item, get_item_bag, get_item_storage
+from modules.tasks import task_is_active
+
+
+def player_pc_menu_index() -> int | None:
+    match context.rom.game_title:
+        case "POKEMON RUBY" | "POKEMON SAPP":
+            raise NotImplementedError("Player PC menu index is not implemented for ruby and sapphire")
+        case "POKEMON EMER":
+            symbol_name = "gPlayerPCItemPageInfo"
+        case "POKEMON FIRE" | "POKEMON LEAF":
+            symbol_name = "gPlayerPcMenuManager"
```

Comment:
You could also use the functions here like `is_emerald`, `ìs_ruby` from `roms.py`

Reply:
Fixed